README.rst

```diff
@@ -100,46 +100,39 @@ September 2023. <https://ieeexplore.ieee.org/abstract/document/10242251>`_
 
 Let us know if you are using snnTorch in any interesting work, research or blogs, as we would love to hear more about it! Reach out at snntorch@gmail.com.
 
-Requirements 
+Installation
 ^^^^^^^^^^^^^^^^^^^^^^^^
-PyTorch should be installed to use snnTorch. Ensure the correct version of torch is installed for your system to enable CUDA compatibility.
-
-The following packages are automatically installed if using the pip command:
 
-* numpy
-* pandas
+Install from PyPI:
 
-The following packages are required for using `export_nir` and `import_nir`:
+.. code-block:: bash
 
-* nir>=1.0.6
-* nirtorch>=2.0.5
+  $ pip install snntorch
 
-The following packages are required for using `spikeplot`:
+Install from source:
 
-* matplotlib
+.. code-block:: bash
 
-Installation
-^^^^^^^^^^^^^^^^^^^^^^^^
+  $ git clone https://github.com/jeshraghian/snnTorch
+  $ cd snntorch
+  $ uv sync
 
-Run the following to install:
+**Note:** For CUDA support, you may need to install PyTorch with pip first:
```

Comment:
Tthe `uv` team is working on better CUDA detection: https://docs.astral.sh/uv/guides/integration/pytorch/#automatic-backend-selection